src/locales/en/messages.json

```diff
@@ -2991,7 +2991,7 @@
     "message": "Require users to set up two-step login on their personal accounts."
   },
   "twoStepLoginPolicyWarning": {
-    "message": "Organization members who do not have two-step login enabled for their personal account will be removed from the organization and will receive an email notifying them about the change."
+    "message": "Organization Users and Managers who do not have two-step login enabled for their personal account will be removed from the organization and will receive an email notifying them about the change."
```

Comment:
Consider rewording: `"Organization members who are not Owners or Administrators and do not have two-step login..."`

To manage by exception vs. full inclusion.

Reply:
Done.